docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.4"
 
 services:
     shipmonk-packing-mysql:
```

Comment:
Let's add 
```
            depends_on:
              - shipmonk-packing-mysql
```
to both adminer & packing app

That does not yet mean that the DB is ready and accepting connections but I'd not bother adding wait-for-it.sh or something similar here yet